_api-reference/profile.md

```diff
@@ -0,0 +1,448 @@
+`score` | Contains the time taken for a Scorer to score a particular document.
+`<method>_count` | Contains the number of invocations of a `<method>`. For example, `advance_count` contains the number of invocations of the `advance` method. Different invocations of the same method happen because the method is called on different documents. You can determine the selectivity of a query by comparing counts in different query components.
+
+### The `collector` object
+
+The `collectors` object contain information about Lucene Collectors. A Collector is responsible for coordinating document traversal and scoring and collecting matching documents. Using Collectors, individual queries can record aggregation results and execute global queries or post-query filters. 
+
+Field | Description
+:--- | :--- 
+`name` | The collector name. In the [example response](#example-response), the `collector` is a single `SimpleTopScoreDocCollector`---the default scoring and sorting collector.
+`reason` | Contains the description of the collector. For possible values of this field, see [Collector reasons](#collector-reasons).
+`time_in_nanos` | A wall-clock time, including timing for all children.
+`children` | A list of subcollectors.
+
+Collector times are calculated, combined, and normalized independently, so they are independent of query times.
+{: .note}
+
+#### Collector reasons
+
+The following table describes all available collector reasons.
+
+Reason | Description
+:--- | :--- 
+`search_sorted` | A collector that scores and sorts documents. Present in most simple searches.
+`search_count` | A collector that counts the number of matching documents but does not fetch the source. Present when `size: 0` is specified.
+`search_terminate_after_count` | A collector that searches for matching documents and terminates the search after it finds a specified number of documents. Present when the `terminate_after_count` query parameter is specified.
+`search_min_score` | A collector that returns matching documents that have a score greater than a minimum score. Present when the `min_score` parameter is specified.
+`search_multi` | A wrapper collector for other collectors. Present when search, aggregations, global aggregations, and post filters are combined in a single search.
+`search_timeout` | A collector that stops executing after a specified period of time. Present when a `timeout` parameter is specified.
+`aggregation` | A collector for aggregations that is run against the specified query scope. OpenSearch uses a single `aggregation` collector to collect docuemnts for all aggregations.
+`global_aggregation` | A collector that is run against the global query scope. Global scope is different from a specified query scope so in order to collect the entire dataset a `match_all` query must be run.
+
+## Aggregations
+
+To profile aggregations, send an aggregation request and provide the `profile` parameter set to `true`.
```

Comment:
We should also include a `Field | Description` section for aggregations and the aggregations `breakdown` object.

Reply:
The aggregations breakdown object has types as below:
```
    INITIALIZE,
    BUILD_LEAF_COLLECTOR,
    COLLECT,
    POST_COLLECTION,
    BUILD_AGGREGATION,
    REDUCE
```